Aptoide/aptoide-client-v8
# Hack/AS-122 add reactions ui

**What does this PR do?**

   Adds reactions UI to editorial cards with fake returns

**Database changed?**

 No

**Where should the reviewer start?**


**How should this be manually tested?**

  Check the fake manager and model and see if the UI behaves according to what’s passed

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/AS-122


**Code Review Checklist**
